dags/dag_content/tower_metrics_content.py

```diff
@@ -24,6 +24,40 @@
 
 # query dictionary - maps query names to queries
 QUERY_DICT = {
+    "total_workflows_per_workspace_past_week": """
+            select workspace, total_runs from (
+            select 
+            w.full_name as workspace, 
+            count(*) AS total_runs 
+            from 
+            tw_workflow wf 
+            join tw_workspace w on wf.workspace_id = w.id
+            where 
+            wf.date_created >= date_sub(now(), interval 1 week) 
+            group by 
+            w.full_name) as total_workflow_runs;
+    """,
+    "workflow_distribution_past_week": """
+            select workspace, workflow, runs, run_distribution from (
+            select
+            w.full_name as workspace,
+            wf.project_name as workflow,
+            count(*) AS runs,
+            count(*) / t.total_runs * 100 as run_distribution
```

Comment:
Nit: Since everything else in the output is "percentage", I vote for consistency.
```suggestion
            count(*) / t.total_runs * 100 as percentage
```

Reply:
addressed in latest commit